testing/regress/ecl/parquetSize.ecl

```diff
@@ -19,6 +19,7 @@
 //nokey
```

Comment:
parquetSize isn't checking the output because it is marked //nokey. Also, I don't think the //no<cluster_name> does anything. It should be either //nothor or //noroxie or //skip